.github/workflows/jreleaser.yml

```diff
@@ -0,0 +1,113 @@
+      - name: Get version with snapshot
+        run: echo "CURRENT_VERSION_WITH_SNAPSHOT=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_ENV
+      # Calculate next version:
+      # - if version is patch, we just increment the patch version
+      # - if version is minor or major, we increment the minor or major version and set the patch version to 0
+      # As we are using a snapshot version, we need to run semver next twice to get the next version for mahor and minor releases. Reason: Any X.Y.Z-SNAPSHOT version will be released as X.Y.Z if we run semver next major X.Y.Z-SNAPSHOT
+      - name: Set next version
+        if: ${{ github.event.inputs.version == 'patch' }}
+        run: echo "NEXT_VERSION=$(semver next ${{ github.event.inputs.version }} $CURRENT_VERSION_WITH_SNAPSHOT)" >> $GITHUB_ENV
+      - name: Set next version
+        # semver next for a snapshot only releases the snapshot version, so we need to run it 2 times
+        if: ${{ github.event.inputs.version == 'major'  || github.event.inputs.version == 'minor' }}
+        run: echo "NEXT_VERSION=$(semver next ${{ github.event.inputs.version }} $CURRENT_VERSION)" >> $GITHUB_ENV
+      - name: Set release version
+        run: mvn --no-transfer-progress --batch-mode versions:set -DnewVersion=$NEXT_VERSION -DprocessAllModules
+      - name: Commit & Push changes
+        uses: actions-js/push@master
+        with:
+          github_token: ${{ secrets.JRELEASER_GITHUB_TOKEN }}
+          message: 'chore: releasing sorald ${{ env.NEXT_VERSION }}'
+
+      # Now we can run the release
+      - name: Stage release
+        run:  mvn --no-transfer-progress --batch-mode -Prelease clean deploy -DaltDeploymentRepository=local::default::file://`pwd`/target/staging-deploy
+      - name: Print next version
+        run: mvn help:evaluate -Dexpression=project.version -q -DforceStdout | sed 's/-SNAPSHOT//'
+      - name: Run JReleaser
+        uses: jreleaser/release-action@v2
+        with:
+          setup-java: false
+          version: 1.4.0
+          arguments: full-release
+        env:
+          JRELEASER_PROJECT_VERSION: ${{ env.NEXT_VERSION }}
+          JRELEASER_GITHUB_TOKEN: ${{ secrets.JRELEASER_GITHUB_TOKEN }}
+          JRELEASER_GPG_PASSPHRASE: ${{ secrets.SORALD_SIGNING_KEY_PASSPHRASE }}
+          JRELEASER_GPG_PUBLIC_KEY: ${{ secrets.SORALD_SIGNING_PUBLIC_KEY }}
+          JRELEASER_GPG_SECRET_KEY: ${{ secrets.SORALD_SIGNING_KEY }}
+          JRELEASER_NEXUS2_MAVEN_CENTRAL_USERNAME: ${{ secrets.OSSH_USERNAME }}
+          JRELEASER_NEXUS2_MAVEN_CENTRAL_PASSWORD: ${{ secrets.OSSH_PASSWORD }}
+
+      # Now we can update the changelog
+      - name: Prepend changelog from out/jreleaser/CHANGELOG.md to CHANGELOG.md with version as header
+        run: |
+          echo "" | cat - CHANGELOG.md > temp && mv temp CHANGELOG.md
```

Comment:
FWIW latest JReleaser offers an option to append the generated changelog to a given file. See `append` section in https://jreleaser.org/guide/latest/reference/release/changelog.html

Reply:
Where can I find the default template `src/jreleaser/templates/append-changelog.tpl`? It does not seem to be hosted on https://github.com/jreleaser/jreleaser.